version: "3"

services:
  databackup:
    image: ferencattila/databackup:latest
    volumes:
      - ./docker_resources/backup:/etc/app/data-backup/backup
      - ./docker_resources/data:/etc/app/data-backup/data
      - ./docker_resources/logs:/etc/app/data-backup/logs
    restart: on-failure